// ng2-charts/esm2015/public_api.js
/**
 * @fileoverview added by tsickle
 * Generated from: public_api.ts
 * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingRequire,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
/*
 * Public API Surface of ng2-charts
 */
export { ChartsModule } from './lib/charts.module';
export { BaseChartDirective } from './lib/base-chart.directive';
export {} from './lib/color';
export {} from './lib/colors';
export { defaultColors } from './lib/default-colors';
export { ThemeService } from './lib/theme.service';
export { monkeyPatchChartJsLegend } from './lib/monkey-patch-chart-js-legend';
export { monkeyPatchChartJsTooltip } from './lib/monkey-patch-chart-js-tooltip';
export { AngularChart } from './lib/chartjs/angular-chart';
export {} from './lib/chartjs/chart-options';
export {} from './lib/chartjs/radial-chart-options';
export {} from './lib/chartjs/chart-type';
export {} from './lib/chartjs/chart-data-sets-base';
export {} from './lib/chartjs/chart-data-sets-bar';
export {} from './lib/chartjs/chart-data-sets-bubble';
export {} from './lib/chartjs/chart-data-sets-doughnut';
export {} from './lib/chartjs/chart-data-sets-line';
export {} from './lib/chartjs/chart-data-sets-radar';
export {} from './lib/chartjs/chart-data-sets-scatter';
export {} from './lib/chartjs/chart-data-sets-union';
export { BaseChartMetaConfig } from './lib/chartjs/base-chart-meta-config';
export { ChartMetaConfig } from './lib/chartjs/chart-meta-config';
export {} from './lib/chartjs/legacy-meta-config';
export {} from './lib/chartjs/promiscuous-meta-config';
export {} from './lib/chartjs/linear-scale';
export {} from './lib/chartjs/logarithmic-scale';
export {} from './lib/chartjs/category-scale';
export {} from './lib/chartjs/time-scale';
export {} from './lib/chartjs/cartesian-scale';
export {} from './lib/chartjs/radial-scale';
export {} from './lib/chartjs/radial-linear-scale';
export {} from './lib/chartjs/scale-union';
